homeassistant/components/motion_blinds/__init__.py

```diff
@@ -161,7 +161,7 @@ def stop_motion_multicast(event):
     device_registry.async_get_or_create(
         config_entry_id=entry.entry_id,
         connections={(dr.CONNECTION_NETWORK_MAC, motion_gateway.mac)},
-        identifiers={(DOMAIN, entry.unique_id)},
+        identifiers={(DOMAIN, entry.unique_id)} if entry.unique_id else None,
```

Comment:
`entry.unique_id` can be `None`. Is this the best way to handle it?

Reply:
If we know that the unique_id is not None, I'd assert that.

Reply:
I've added an assert before `async_get_or_create`